From: Tom Gundersen <teg@jklm.no>
Date: Fri, 4 Jul 2014 01:26:19 +0200
Subject: man: netdev - mention tun and tap

Reported by Moviuro <moviuro@gmail.com>

(cherry picked from commit 54f601debc07addbed803fb847cd5bd3e91e021e)
---
 man/systemd.netdev.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/man/systemd.netdev.xml b/man/systemd.netdev.xml
index c90bd8f..857221f 100644
--- a/man/systemd.netdev.xml
+++ b/man/systemd.netdev.xml
@@ -163,7 +163,8 @@
                                                 <literal>macvlan</literal>, <literal>vxlan</literal>,
                                                 <literal>ipip</literal>, <literal>gre</literal>,
                                                 <literal>sit</literal>, <literal>vti</literal>,
-                                                <literal>veth</literal>, and <literal>dummy</literal>
+                                                <literal>veth</literal>, <literal>tun</literal>,
+                                                <literal>tap</literal> and <literal>dummy</literal>
                                                 are supported. This option is compulsory.</para>
                                         </listitem>
                                 </varlistentry>
